specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -1980,6 +1980,21 @@
         }
       }
     },
+    "AzureMySqlSink": {
```

Comment:
>AzureMySqlSink [](start = 5, length = 14)

Is anything referencing this?

Reply:
This is a new Sink type that is added, as similar as other sinks.